packages/auditd_manager/manifest.yml

```
@@ -1,15 +1,15 @@
 format_version: "3.0.0"
 name: auditd_manager
 title: "Auditd Manager"
-version: "1.16.4"
+version: "1.17.0"
 description: "The Auditd Manager Integration receives audit events from the Linux Audit Framework that is a part of the Linux kernel."
 type: integration
 categories:
   - security
   - auditd
 conditions:
   kibana:
-    version: "^8.7.1"
+    version: "^8.11.0"
```

```suggestion
    version: "^8.12.0"
```

Let's try 8.12.0 and see if CI is still failing.